pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  makeWrapper,
+}:
+let
+  pythonEnv = python3.buildEnv.override {
+    extraLibs = with python3.pkgs; [
+      tkinter
+      requests
+      pillow
+      (watchdog.overrideAttrs {
+        disabledTests = [
+          "test_select_fd"
+        ];
+      })
+      semantic-version
+      psutil
+    ];
+    ignoreCollisions = true;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "edmarketconnector";
+  version = "5.13.1";
+
+  src = fetchFromGitHub {
+    owner = "EDCD";
+    repo = "EDMarketConnector";
+    tag = "Release/${version}";
+    hash = "sha256-50OPbAXrDKodN0o6UibGUmMqQ/accF2/gNHnms+8rOI=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstallPhase
+
+    install -Dm644 ${src}/io.edcd.EDMarketConnector.png $out/share/icons/hicolor/512x512/apps/io.edcd.EDMarketConnector.png
```

Comment:
nit: as we are referencing `src` in the final drv anyway, we could probably just symlink this

Reply:
Should I make a symlink or have the app in $out as well?

Reply:
Not sure if we have conventions for projects like this. It might be possible to package the app using `buildPythonPackage` even

Reply:
> It might be possible to package the app using buildPythonPackage even

Not that i know of. It doesn't use either of the 2 ways `buildPythinPackage` works to build Python apps.

If you know how I can rewrite it I will do so, but I asked on Discord when I was unsuccessful with `buildPythonPackage` and this is what people came up with.